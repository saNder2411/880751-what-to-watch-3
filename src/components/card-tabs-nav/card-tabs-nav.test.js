import React from 'react';
import renderer from 'react-test-renderer';
import CardTabsNav from './card-tabs-nav';

const cardMode = `overview`;
const handleTabsNavClick = () => {};

it(`Should CardOverview render correctly`, () => {
  const markup = renderer
    .create(<CardTabsNav mode={cardMode} onTabsNavClick={handleTabsNavClick} />)
    .toJSON();

  expect(markup).toMatchSnapshot();
});
