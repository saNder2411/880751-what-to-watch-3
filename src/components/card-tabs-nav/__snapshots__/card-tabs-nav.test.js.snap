// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should CardOverview render correctly 1`] = `
<nav
  className="movie-nav movie-card__nav"
>
  <ul
    className="movie-nav__list"
    onClick={[Function]}
  >
    <li
      className="movie-nav__item movie-nav__item--active"
    >
      <a
        className="movie-nav__link"
        href="#"
      >
        Overview
      </a>
    </li>
    <li
      className="movie-nav__item "
    >
      <a
        className="movie-nav__link"
        href="#"
      >
        Details
      </a>
    </li>
    <li
      className="movie-nav__item "
    >
      <a
        className="movie-nav__link"
        href="#"
      >
        Reviews
      </a>
    </li>
  </ul>
</nav>
`;
